ui/next.config.js

```diff
@@ -1,4 +1,18 @@
 /** @type {import('next').NextConfig} */
+const CSPDirectives = [
+    "default-src 'self';",
```

Comment:
Wouldn't this be better as 'none'? Ideally everything will be set to something or 'self', everything else 'none'.

Reply:
Yeah, it makes sense for the catch-all to be less permissive. I've set it to 'none'